### PR TITLE
Parse bpf loader instructions

### PR DESCRIPTION
**Pull request by CriesofCarrots**
#### Problem
The BPF Loader instructions show up as Raw in the explorer.

#### Summary of Changes
- Add json parsing for LoaderInstructions

Toward #12812

```json
// Write example ix

"instructions": [
    {
      "parsed": {
        "info": {
          "bytes": "AAAAAAAAAAABAAAAAAAAAAAAAABgrAEAAAAAAOAnAgAgAAAAAAAAAAAAAABbQwIAEgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAHAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAbAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAmAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAEgABAMj2AAAAAAAAgAEAAAAAAAAAYWJvcnQAc29sX2xvZ18AZW50cnlwb2ludABzbw==",
          "offset": 152056
        },
        "type": "write"
      },
      "program": "bpf-loader",
      "programId": "BPFLoader2111111111111111111111111111111111"
    }
  ]

// Finalize example ix

"instructions": [
    {
      "parsed": {
        "type": "finalize"
      },
      "program": "bpf-loader",
      "programId": "BPFLoader2111111111111111111111111111111111"
    }
  ]
```

### CODE REVIEW

**Comment by CriesofCarrots** [comment]
@oJshua can you please take a look and see if this will suit you for #12812 ? In particular:
- do you need the destination account address in the `info` map?
- any reason not to return `bytes` as base64-encoded string?

**Comment by oJshua** [comment]
> * do you need the destination account address in the `info` map? ... thinking on this more, I think it will be more consistent with parse_token if we do include this, so I will make that change, unless you object

No objection here!
> * any reason not to return `bytes` as base64-encoded string?

Works for me.
> Also, let me know if we need to return `info: null` instead of skipping that field when null

Skipping will be just fine!

This all works perfectly for #12812 . Thanks!

**Review by oJshua — APPROVED** [review]

File: transaction-status/src/parse_bpf_loader.rs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    instruction::CompiledInstruction, loader_instruction::LoaderInstruction, pubkey::Pubkey,
+};
+
+pub fn parse_bpf_loader(
+    instruction: &CompiledInstruction,
+    account_keys: &[Pubkey],
+) -> Result<ParsedInstructionEnum, ParseInstructionError> {
+    let bpf_loader_instruction: LoaderInstruction =
+        deserialize(&instruction.data).map_err(|err| {
+            println!("{:?}", err);
```
Comment (mvines) [inline transaction-status/src/parse_bpf_loader.rs:14]: 🖨️

Comment (CriesofCarrots) [reply transaction-status/src/parse_bpf_loader.rs:14]: 🤦‍♀️

**Pull request merged**